### PR TITLE
Add dbt test and dbt build to fail fast

### PR DESCRIPTION
**Pull request by genwhitt**
## What are you changing in this pull request and why?
Add dbt test and dbt build to fail fast (fix for issue https://github.com/dbt-labs/docs.getdbt.com/issues/911).

### CODE REVIEW

File: website/docs/reference/global-configs.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ config:
 
 ### Failing fast
 
-Supply the `-x` or `--fail-fast` flag to `dbt run` to make dbt exit immediately if a single resource fails to build. If other models are in-progress when the first model fails, then dbt will terminate the connections for these still-running models.
+[Supply the `-x` or `--fail-fast` flag to `dbt run` to make dbt exit immediately if a single resource fails to build. If other models are in-progress when the first model fails, then dbt will terminate the connections for these still-running models.](https://github.com/dbt-labs/docs.getdbt.com/issues/911)
```
Comment (nghi-ly) [inline website/docs/reference/global-configs.md]: hi @genwhitt ! thanks for visiting our table at WTD Writing Day. I think the copy/pasta we were working on got mixed up. Let's get your real update in!

**Review by nghi-ly — COMMENTED** [review]
Let's get your real contribution in!

**Review by nghi-ly — COMMENTED** [review]
Let's get your real contribution in!

File: website/docs/reference/global-configs.md (hunk 1 above)
Comment (genwhitt) [inline website/docs/reference/global-configs.md]:
```suggestion
Supply the `-x` or `--fail-fast` flag to `dbt run`, `dbt test`, or `dbt build` to make dbt exit immediately if a single resource fails to build. If other models are in-progress when the first model fails, dbt terminates the connections for these models that are still running.
```

**Review by nghi-ly — APPROVED** [review]
amazing, thank you so much!

**Pull request merged**